.env.default

```diff
@@ -17,10 +17,10 @@ DEV_MODE=true
 LAGOON_ENVIRONMENT_TYPE=local
 
 # https://govcms.gov.au/wiki_vars#LOCALDEV_URL
-LOCALDEV_URL=http://govcms8-paas.docker.amazee.io
+LOCALDEV_URL=http://govcms8-scaffold-satis.docker.amazee.io
 
 # https://govcms.gov.au/wiki_vars#MARIADB_DATA_IMAGE
-#MARIADB_DATA_IMAGE=gitlab-registry-production.govcms.amazee.io/org/project/mariadb-drupal-data
+MARIADB_DATA_IMAGE=gitlab-registry-production.govcms.amazee.io/dof-dev/govcms8-scaffold-satis/mariadb-drupal-data
```

Comment:
* Should we be advertising Finance (non-customer-facing) namespaces?
* Should PaaS users be able to access this image with their existing credentials?
* Is Satis formally no longer a POC?

Reply:
Shouldn't be here. I just pushed to revert this.